### DISCUSSION TITLE
dbeaver causing error: too many open files

### DISCUSSION THREAD

**Discussion by erlangparasu** [dbeaver/dbeaver#13683] [Q&A]
i need to restart mysql server to fix this problem.

**Comment by uslss** [comment]
@erlangparasu please provide steps to reproduce.
Also, you could try to disable Open separate connection for each editor and Open separate connection for metadata read options in preferences
![image](https://user-images.githubusercontent.com/31996417/127830988-1e291681-0094-43f9-b09b-13f150056abe.png)
![image](https://user-images.githubusercontent.com/31996417/127831119-49d79006-2b9e-4c04-aaef-faa4d5e2f960.png)

**Comment by SubOptimal** [comment]
@erlangparasu Which OS are you using?

On Linux, you could check how many files a process can open at a time with `ulimit -n`.
